charts/ceph-csi-cephfs/README.md

```diff
@@ -148,18 +148,21 @@ charts and their default values.
+| `provisioner.provisioner.args.httpEndpoint`    | Specifies httpEndpoint for diagnostics, health checks and metrics                                                                                    | `""`                                               |
+| `provisioner.provisioner.extraArgs`            | Specifies extra arguments for the provisioner sidecar                                                                                                | `[]`                                               |
+| `provisioner.resizer.name`                     | Specifies the name of csi-resizer sidecar                                                                                                            | `resizer`                                          |
+| `provisioner.resizer.enabled`                  | Specifies whether resizer sidecar is enabled                                                                                                         | `true`                                             |
 | `provisioner.resizer.image.repository`         | Specifies the csi-resizer image repository URL                                                                                                       | `registry.k8s.io/sig-storage/csi-resizer`          |
 | `provisioner.resizer.image.tag`                | Specifies image tag                                                                                                                                  | `v1.11.1`                                           |
 | `provisioner.resizer.image.pullPolicy`         | Specifies pull policy                                                                                                                                | `IfNotPresent`                                     |
-| `provisioner.resizer.image.extraArgs`          | Specifies extra arguments for the resizer sidecar                                                                                                                                | `[]`                                     |
-| `provisioner.resizer.name`                     | Specifies the name of csi-resizer sidecar                                                                                                            | `resizer`                                          |
-| `provisioner.resizer.enabled`                  | Specifies whether resizer sidecar is enabled                                                                                                         | `true`                                             |
+| `provisioner.resizer.args.httpEndpoint`        | Specifies httpEndpoint for diagnostics, health checks and metrics                                                                                    | `""`                                               |
```

Comment:
This isn't really the endpoint, it is only a port. The whole endpoint gets combined with the IP-address of the pod. The description should be improved, and maybe rename `.httpEndpoint` to `.httpEndpointPort`.

Reply:
Makes sense, updated the doc and the variable name.